docs/yml.md

```diff
@@ -114,6 +114,14 @@ An entry with the ip address and hostname will be created in `/etc/hosts` inside
 50.31.209.229   otherhost
 ```
 
+### read_only (bool)
+
+mount the container's root filesystem as read only. Same as `--read-only`
+
+```
+read_only: true
+```
+
```

Comment:
Since this is a simple single-value option, it can go in the big catch-all section at the bottom.